code/game/objects/items/antiques.dm

```diff
@@ -0,0 +1,209 @@
+	..()
+	switch(rand(1,3))
+		if (1)
+			name = "Locket"
+			desc = "A precious container for someone's cherished memories."
+			icon_state = "alocket"
+		if (2)
+			name = "Homeworld"
+			desc = "A miniature representation of someone's home planet."
+			icon_state = "aplanet"
+		if (3)
+			name = "Porcelain Doll"
+			desc = "A precious porcelain doll resembling an harlequin."
+			icon_state = "adoll"
+
+/obj/item/artifact/posessed/Initialize()
+	..()
+	if (prob(33))
+		deity = rand(1,GODS_MAX)
+		switch (deity)
+			if (GOD_NETHER)
+				godname = "Lobon"
+			if (GOD_SIGHT)
+				godname = "Nath-Horthath"
+			if (GOD_STONE)
+				godname = "Oukranos"
+			if (GOD_CLEANSE)
+				godname = "Tamash"
+			if (GOD_MEND)
+				godname = "Karakal"
+			if (GOD_CAUTERIZE)
+				godname = "D’endrrah"
+			if (GOD_MUTE)
+				godname = "Abhoth"
+			if (GOD_FORCE)
+				godname = "Ialdagorth"
+			if (GOD_PARALIZE)
+				godname = "C'thalpa"
+			if (GOD_ELDRITCH)
+				godname = "Yomagn'tho"	
+
+/obj/item/artifact/examine(mob/user)
+	. = ..()
+	if (deity == NO_GOD)
+		return .
+	var/boon_type = deity < GOD_ELDRITCH ? "curse" : "blessing"
+	
+	var/mob/living/carbon/C = user
+	var/datum/antagonist/heretic/heretic_user = user.mind.has_antag_datum(/datum/antagonist/heretic)	
+	if (heretic_user || (istype(C) && C.job == "Curator"))		
+		if (activated)
+			var/boon = "The [name] bestows the [boon_type] of [godname], "			
+			switch (deity)
+				if (GOD_MEND)
+					boon += "mending burns on the target."
+				if (GOD_CAUTERIZE)
+					boon += "healing bruises on the target."
+				if (GOD_CLEANSE)
+					boon += "flushing toxins from the target."
+				if (GOD_NETHER)
+					boon += "sending a hostile creature on the target."
+				if (GOD_SIGHT)
+					boon += "blinding the target."
+				if (GOD_STONE)
+					boon += "petrifying the target."
+				if (GOD_MUTE)
+					boon += "preventing the target from speaking"
+				if (GOD_FORCE)
+					boon += "pushing the target away."
+				if (GOD_PARALIZE)
+					boon += "paralizing the target."
+				if (GOD_ELDRITCH)
+					boon += "healing Heretics and damages others."				
+			. += boon
+		else if (heretic_user)
+			. += "Use it while holding a Codex Cicatrix in your other hand to perform a ritual of admration for [godname], and infuse this [src] with a magical effect."			
+		if (!heretic_user?.has_deity(deity))
+			. += "Performing a ritual of admration for [godname] will grant you a charge to your Codex Cicatrix."
+
+/obj/item/artifact/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	..()
+	if(world.time > last_use && proximity_flag)
+		to_chat(user,"<span class='warning'>You try to hex [target] with the [src]!</span>")
+		if (HAS_TRAIT(target,TRAIT_WARDED))
+			to_chat(user,"<span class='warning'>[target] is warded against your hex!</span>")
+			to_chat(target,"<span class='warning'>Your crucifix protects you against [user]'s hex!</span>")
+		else 
+			infuse_blessing(user,target)
+		last_use = world.time + 300 SECONDS
```

Comment:
these have no cost other than only being able to use them every 5 minutes? (Also use 5 MINUTES define instead of seconds)

Reply:
Yes